fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -428,6 +435,15 @@ protected FedoraResource resource() {
         return resource;
     }
 
+    /**
+     * Add the standard Accept-Post header, for reuse.
+     */
+    protected void addAcceptPostHeader() {
+        final String rdfTypes = TURTLE + "," + N3 + "," + N3_ALT2 + "," + RDF_XML + "," + NTRIPLES + "," + JSON_LD;
+        servletResponse.addHeader("Accept-Post", rdfTypes + "," + MediaType.MULTIPART_FORM_DATA + "," +
```

Comment:
Although support for `multipart/form-data` makes sense on POST, I do not see it anywhere else in the codebase.